### DISCUSSION TITLE
Roles & Skills on BOLT🔩FUN

### DISCUSSION THREAD

**Discussion by steady-eddie** [peakshift/makers.bolt.fun#143] [Ideas]
Would you like to see another Role or Skill featured on BOLT🔩FUN? If so, you can comment on this discussion below with your idea. 

If you'd like to add a new item, you can use this template:

**Item type** : Role/Skill
**Item name**: e.g Developer
**Reasoning**: (if any)

Here is a current list of the Roles & Skills...

## Roles 🎛

> - 💄 Frontend Dev
> - 💻 Backend Dev
> - 🌈 UI/UX Designer
> - 🎉 Community Manager
> - 🦄 Founder
> - 🚨 Marketer
> - 🎥 Content Creator
> - 🔬 Researcher
> - 💿 Data Engineer
> - 📈 Growth Hacker
> - ✍️ Writer

### Design
> - Wireframing
> - Prototyping
> - UI design
> - UX design
> - User testing
> - User research
> - Illustration
> - Graphic design
> - Figma
> - Principle
> - Miro
> - Website design
> - Logo design

### Engineering
> - HTML/CSS
> - JavaScript
> - TypeScript
> - React.Js
> - Vue.Js
> - Angular.Js
> - jQuery
> - Wordpress
> - GraphQl
> - Node.Js
> - PHP
> - .Net
> - Python
> - Java
> - C++
> - SQL
> - NoSQL
> - Git
> - Testing

### Product Management
> - Github
> - Trello
> - ClickUp
> - Storybook
> - Task management

**Comment by steady-eddie** [comment]
**Request**

Item type : Role
Item name: 🧿 Product Manager
Reasoning: Can't believe I forgot this one...

Item type : Skill
Item name: Roadmapping
Reasoning: An important role for PMs

**Reply by sebrock** [reply]
I dearly miss this.
Strong vote. 🚀

**Comment by Amyyalby** [comment]
Request 

Item Type: Marketing 
Item Name: Social Media Manager
Reasoning - everyone needs a SMM these days.

**Reply by steady-eddie** [reply]
@Amyyalby what's the difference between a social media manager and community manager?

**Comment by nully0x** [comment]
Request

Item Type: Skill/Engineering
Item Name: DevOps
Reasoning -This is an important role for Cloud Engineer/Developer.

Item Type: Skill/Engineering
Item Name: Rust
Reasoning -This is an important role for a Backend Engineer.

**Reply by steady-eddie** [reply]
Added DevOps as a role @nully0x ! Also Rust as a skill 👌

**Comment by saentari** [comment]
Request

Item Type: Skill/Engineering
Item Name: **Flutter**
Reasoning: This is an important skill for Developers using the Flutter framework.

Item Type: Skill/Engineering
Item Name: **Dart**
Reasoning: This is an important skill for Developers programming in the Dart language.

**Reply by steady-eddie** [reply]
Added x2 skills!

**Comment by johnsBeharry** [comment]
Item Type: Skill/Engineering
Item Name: WebBTC / WebLN
Reasoning: ...

Item Type: Skill/Engineering
Item Name: LNURL-Channel
Reasoning: ...

Item Type: Skill/Engineering
Item Name: LNURL-Pay
Reasoning: ...

Item Type: Skill/Engineering
Item Name: LNURL-Withdraw
Reasoning: ...

Item Type: Skill/Engineering
Item Name: Nostr
Reasoning: ...

Item Type: Skill/Engineering
Item Name: Node Management
Reasoning: Plebs and devs be managing nodes.

**Comment by nade-harlow** [comment]
Request

Item Type: Skill/Engineering
Item Name: Go
Reasoning: This is the cool kid in the block.

**Comment by BitMaker-hub** [comment]
Request

Item type : Role
Item name: ⛏ **Hardware Manager**
Reasoning: I can't fit on any Role. I make hardware, and lightning or Bitcoin is completely connected to HW.

Item type : Skill
Item name: **Arduino**
Reasoning: This is an important skill for those who want a fast prototyping

Item type : Skill
Item name: **PlatformIO**
Reasoning: This is an important skill for those who want a fast prototyping

**Reply by johnsBeharry** [reply]
@BitMaker-hub nice suggestions! We'll get these added. I really want to see some hardware being built during the tournament!

**Reply by steady-eddie** [reply]
Done! 🎉

**Comment by Giszmo** [comment]
Engineering skills missing:

* Svelte (Like Vue)
* Kotlin (Android switched completely from Java to Kotlin)

**Reply by steady-eddie** [reply]
Thank you, sir. Added 🤝

**Comment by guiporto** [comment]
Request

Item Type: Sales
Item Name: Sales/Business Development
Reasoning: Important role specially for those aiming B2B

**Reply by steady-eddie** [reply]
I've added Business Dev 🤝 as there seem to be a lot on these in the space!

**Comment by olawalethefirst** [comment]
Request

Item Type: Skill/Engineering
Item Name: React Native
Reasoning: This is an important skill for Developers using the React Native framework.

**Comment by LatiNodos** [comment]
Request

Item Type: Skill/Content Creator
Item Name: Audio/Video editing
Reasoning: Using software/platforms to edit and create quality digital content

**Comment by jacobpiotrowski** [comment]
Request

Item Type: Skill/Scrum Master
Item Name: Jira/Scrum
Reasoning: This is an important role within a distributed dev team.

**Comment by aljazceru** [comment]
Request

Item Type: Role
Item Name: Architect 

Item Type: Skill
Item Name: Linux

Item Type: Skill
Item Name: System Administration

Item Type: Skill
Item Name: Security

**Reply by johnsBeharry** [reply]
Added!

**Comment by bitcoinphotographer** [comment]
Request

Item type: UX/UI Designer
Item name: User Personas
Reasoning: To visualize who we are building for and understand their motivations, etc.

Item type : UX/UI Designer
Item name: Journey Mapping
Reasoning: Need to understand the experience of our persona and determine at what point we are going to intervene.

**Reply by johnsBeharry** [reply]
Added!

**Comment by ghost** [comment]
Request

Item Type: Engineering
Item Name: Ruby / Ruby On Rails
Reasoning: Popular lang, framework

Item Type: Engineering
Item Name: Elixir
Reasoning: High performance, elegant language

**Reply by johnsBeharry** [reply]
Added!

**Comment by brenorb** [comment]
Item type : Role
Item name: Data Scientist

Item type : Role
Item name: Machine Learning Engineering

**Comment by CMPGFB** [comment]
Item Type: Engineering
Item Name: NoCode
Reasoning: Globally used, great for launching apps and/or MVPs
